src/www/ui/api/documentation/openapi.yaml

```diff
@@ -15,7 +15,7 @@ openapi: 3.0.2
 info:
   title: FOSSology API
   description: Automate your fossology instance using REST API
-  version: 1.2.1
+  version: 1.2.2
```

Comment:
I don't think it is necessary to update the API version. It is just a documentation patch. What do you think?

Reply:
This is a thing I also thought about. Finally I decided to change it as I don't know Your versioning policy very well. 
In fact, last part in semver policy says that last section in version number is for patch purposes, thats why I decided to change it.
All in all, I can revert this change if you wish so.

Reply:
Yeah it is true that following semver policy, it should change to 1.2.2 but for the REST API, we tried to keep it simple.
1. If a change **does not** break backward compatibility (in terms of API call) but adds something new, we change the patch version (1.2.1 => 1.2.2).
2. If a change **does break** backward compatibility, we like to make it explicit by changing the minor version (1.2.1 => 1.3.0).
3. We have not yet faced or decided anything to change the major version of API :neutral_face:

Reply:
So yeah, these changes does not do anything on the API so it should be okay to leave the version as is.

Reply:
Ok, Your explanation is clear.
Update reverting API version change is on it's way